crates/ruff/src/rules/pyflakes/rules/imports.rs

```diff
@@ -157,18 +155,13 @@ impl Violation for LateFutureImport {
 #[violation]
 pub struct UndefinedLocalWithImportStarUsage {
     pub(crate) name: String,
-    pub(crate) sources: Vec<String>,
 }
 
 impl Violation for UndefinedLocalWithImportStarUsage {
     #[derive_message_formats]
     fn message(&self) -> String {
-        let UndefinedLocalWithImportStarUsage { name, sources } = self;
-        let sources = sources
-            .iter()
-            .map(|source| format!("`{source}`"))
-            .join(", ");
```

Comment:
I decided to drop this behavior. I don't think it's super important, and it's harder to support now (unless we're comfortable including imports that take place in the same scope, but _after_ a reference, like `print(x); from foo import *`).